### DISCUSSION TITLE
Cannot set properties of undefined (setting 'textContent') at e.g.mosparo.countDown (mosparo-frontend.js:1:17405)

### DISCUSSION THREAD

**Discussion by raul810y** [mosparo/mosparo#286] [Problems & Issues]
Hi

I'm doing a custom mosparo integration.

When I enable the "security settings" and try to test "Request delaying" and "IP lockout" I see in the web console that it starts to throw the following error repeatedly:

TypeError: Cannot set properties of undefined (setting 'textContent') at e.g.mosparo.countDown (mosparo-frontend.js:1:17405)

I attach a video with the behavior for quick reference:

https://drive.google.com/file/d/1NpZ26W2Xe5iFgHPSVcGSWFjfxfNuOjOA/view?usp=sharing

Mosparo version: 1.2.8

**Comment by zepich** [comment]
Hi @raul810y 

Thank you very much for your detailed report.

I tested the code and was able to reproduce it. I've added a fix, which we will release with v1.2.9 in the next few days.

Thank you for making mosparo better!

Kind regards,

zepich
